pygmt/helpers/utils.py

```diff
@@ -315,6 +315,108 @@ def non_ascii_to_octal(argstr):
     return argstr.translate(str.maketrans(mapping))
 
 
+def build_arg_list(
+    kwdict: dict,
```

Comment:
Mention what's in the `dict`?
```suggestion
    kwdict: dict[str, Any],
```